Src/Compilers/Core/MSBuildTask/NativeMethods.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.CodeAnalysis.BuildTasks
+{
+    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+    internal struct STARTUPINFO
+    {
+        internal Int32 cb;
+        internal string lpReserved;
+        internal string lpDesktop;
+        internal string lpTitle;
+        internal Int32 dwX;
+        internal Int32 dwY;
+        internal Int32 dwXSize;
+        internal Int32 dwYSize;
+        internal Int32 dwXCountChars;
+        internal Int32 dwYCountChars;
+        internal Int32 dwFillAttribute;
+        internal Int32 dwFlags;
+        internal Int16 wShowWindow;
+        internal Int16 cbReserved2;
+        internal IntPtr lpReserved2;
+        internal IntPtr hStdInput;
+        internal IntPtr hStdOutput;
+        internal IntPtr hStdError;
+    }
+
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct PROCESS_INFORMATION
+    {
+        public IntPtr hProcess;
+        public IntPtr hThread;
+        public int dwProcessId;
+        public int dwThreadId;
+    }
+
+    /// <summary>
+    /// Interop methods.
+    /// </summary>
+    internal static class NativeMethods
+    {
+        #region Constants
+
+        internal static readonly IntPtr NullPtr = IntPtr.Zero;
+        internal static readonly IntPtr InvalidIntPtr = new IntPtr((int)-1);
+
+        internal const uint NORMAL_PRIORITY_CLASS = 0x0020;
+        internal const uint CREATE_NO_WINDOW = 0x08000000;
+        internal const Int32 STARTF_USESTDHANDLES = 0x00000100;
+        internal const int ERROR_SUCCESS = 0;
```

Comment:
Not a big issue, but it looks a bit odd having `Int32` on one line, and `int` on the next line.